src/test/resources/features/quiz-quizTitle-neta.feature

```diff
@@ -0,0 +1,178 @@
+    And I add a question
+    And I wait for 1 sec
+    When I select "Single" question type
+    And I type "Question 1 name" into "Q1"
+    When I type "Option 1" as "Option 1*" into "Q1"
+    And I type "Option 2" as "Option 2*" into "Q1"
+    And I select "Option 1*" as correct option in "Q1"
+    When I click button "Save"
+    # This is known issue
+    Then error message should be displayed
+
+
+  @quiz8
+  Scenario: Quiz - Quiz Title with spaces between the characters - Negative
+    When I click button "Create New Quiz"
+    And I wait for 1 sec
+    When I type "bn  bn" as quiz title
+    And I add a question
+    And I wait for 1 sec
+    When I select "Single" question type
+    And I type "Question 1 name" into "Q1"
+    When I type "Option 1" as "Option 1*" into "Q1"
+    And I type "Option 2" as "Option 2*" into "Q1"
+    And I select "Option 1*" as correct option in "Q1"
+    When I click button "Save"
+    And I wait for 2 sec
+    # this is known issue
+    Then error message should be displayed
+
+    @quiz9
+    Scenario: Quiz - Quiz Title with leading characters - Negative
+      When I click button "Create New Quiz"
+      And I wait for 1 sec
+      When I type "  bn bn" as quiz title
+      And I add a question
+      And I wait for 1 sec
+      When I select "Single" question type
+      And I type "Question 1 name" into "Q1"
+      When I type "Option 1" as "Option 1*" into "Q1"
+      And I type "Option 2" as "Option 2*" into "Q1"
+      And I select "Option 1*" as correct option in "Q1"
+      When I click button "Save"
+      And I wait for 2 sec
+    # this is known issue
+      Then error message should be displayed
+
+      @quiz10
+      Scenario: Quiz - Quiz Title with trailing characters - Negative
+        When I click button "Create New Quiz"
+        And I wait for 1 sec
+        When I type "bn bn     " as quiz title
+        And I add a question
+        And I wait for 1 sec
+        When I select "Single" question type
+        And I type "Question 1 name" into "Q1"
+        When I type "Option 1" as "Option 1*" into "Q1"
+        And I type "Option 2" as "Option 2*" into "Q1"
+        And I select "Option 1*" as correct option in "Q1"
+        When I click button "Save"
+        And I wait for 2 sec
+    # this is known issue
+        Then error message should be displayed
```

Comment:
You need to change the wording of this step ( it has different code bu the same name)